schemas/function.yaml

```diff
@@ -163,6 +171,10 @@ $defs:
           description: |
             The default value for this parameter, if none was given in the call to the function.
             This property automatically implicitly marks this parameter as optional.
+        optional:
```

Comment:
As I explained in the other PR #9 where you tried to implement this optional attribute, it is not needed.